rules/promotions/sentinelone_external_alerts.toml

```diff
@@ -0,0 +1,111 @@
+[metadata]
+creation_date = "2025/08/05"
+integration = ["sentinel_one"]
+maturity = "production"
+promotion = true
+min_stack_version = "8.18.0"
+min_stack_comments = "Introduced support for SentinelOne alert promotion"
+updated_date = "2025/08/05"
```

Comment:
Should these updated dates be in the future? It seems like this is incorrect unless I am missing something.